lib/src/base-styles/_nx-typography.scss

```diff
@@ -83,10 +83,9 @@
 }
 
 .nx-label {
-  @include semi-bold();
+  @include bold();
```

Comment:
As I mentioned above, this is out of sync with the styles in `_nx-forms.scss`.  I also wonder why any `.nx-label` styles would be in this file at all given the existence of `_nx-forms.scss`

Reply:
> As I mentioned above, this is out of sync with the styles in `_nx-forms.scss`. I also wonder why any `.nx-label` styles would be in this file at all given the existence of `_nx-forms.scss`

I don't think so, I consolidated it into `forms.scss`: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/1c38ba52ad23e9a7a48948ca4119360923e9fc9a